server/src/main/java/org/opensearch/search/wlm/QueryGroupConstants.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.wlm;
+
+import java.util.function.Supplier;
+
+/**
+ * This class will hold all the QueryGroup related constants
+ */
+public class QueryGroupConstants {
```

Comment:
Stub, this will be introduced into main by this PR - https://github.com/opensearch-project/OpenSearch/pull/14708/files#diff-975b1ca9eecb56f601cef1bd3074b11a9e636463f077f3e72221c19d5f427156R16